book/10-git-internals/sections/packfiles.asc

```diff
@@ -118,8 +119,8 @@ Because you never added them to any commits, they're considered dangling and are
 The other files are your new packfile and an index.
 The packfile is a single file containing the contents of all the objects that were removed from your filesystem.
 The index is a file that contains offsets into that packfile so you can quickly seek to a specific object.
-What is cool is that although the objects on disk before you ran the `gc` were collectively about 22K in size, the new packfile is only 7K.
-You've cut your disk usage by ⅔ by packing your objects.
+What is cool is that although the objects on disk before you ran the `gc` were collectively about 15K in size, the new packfile is only 7K.
+You've cut your disk usage by ½ by packing your objects.
 
```

Comment:
I checked the web page. It's good, not too small. :smile: 

![image](https://cloud.githubusercontent.com/assets/5831804/13169405/2d51ea58-d720-11e5-8369-b6e52fe22971.png)